tests/contrib/starlette/test_starlette.py

```diff
@@ -24,7 +24,7 @@
 
 
 starlette_version_str = getattr(starlette, "__version__", "0.0.0")
-starlette_version = tuple([int(i) for i in starlette_version_str.split(".")])
+starlette_version = tuple([int(i) for i in starlette_version_str.split(".")[:3]])
```

Comment:
Can you explain why this slice is needed?

Reply:
I used a slice to always grab the major, minor and patch version and ignore everything else. So in this case 
0.26.0.post1 -> [0, 26, 0]